sycl/include/sycl/ext/oneapi/experimental/root_group.hpp

```diff
@@ -42,6 +42,13 @@ template <> struct detail::PropertyToKind<use_root_sync_key> {
 template <>
 struct detail::IsCompileTimeProperty<use_root_sync_key> : std::true_type {};
 
+enum class execution_scope {
+  work_item,
+  sub_group,
+  work_group,
+  root_group,
```

Comment:
I'm also wondering if we need this value now, because there are no use-cases. This enum was introduced by another spec, which I used as a reference for https://github.com/intel/llvm/pull/12643, so I preserved it. I'd say we may need it for some future work and to reflect all `execution_scope` hierarchy.